docs/core/usage.qmd

```diff
@@ -42,6 +42,9 @@ geopackage = "model.gpkg"  # required
 flow = "output/flow.arrow"    # optional, default "output/flow.arrow"
 basin = "output/basin.arrow"  # optional, default "output/basin.arrow"
 
+# This output file is only written if the model involves control
```

Comment:
Yeah, so we could have a control table with rules that are never triggered right? This comment implies that we should write an empty table.